<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReserveBook</class>
 <widget class="QDialog" name="ReserveBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yoobee Library System | Reserve a book</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/favicon.ico</normaloff>:/images/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
fontsize: 15px;
font-family: Century Gothic, sans-serif;
}

QPushButton{
color: white;
padding: 5px;
background: #27a9e3;
border-style: outset;
border-radius: 5px;
}
QPushButton::pressed{
background-color: rgb(216, 150, 90);
border-style: inset;
}

QPushButton::hover {
background-color: #59c7f7;
}

QPushButton::disabled {
color: #ebebeb;
background-color: #1c7ba6;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Reserve in Advance</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>You are placing a reservation for </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="bookNameLabel">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&quot;Book Name&quot;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Checkout Date:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Return Date:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="returnDateLabel">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&quot;Return Date&quot;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="confirmReserve">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
